tests/builder/help.rs

```diff
@@ -38,9 +38,11 @@ OPTIONS:
         --multvalsmo <one> <two>     Tests multiple values, and mult occs
     -o, --option <opt>...            tests options
     -O, --option3 <option3>          specific vals [possible values: fast, slow]
-        --optvaleq[=<optval>]        Tests optional value, require = sign
```

Comment:
I think this is a bugfix, since `min_value(0) + number_of_values(1)` makes `--optvaleq` without value incorrect. which means the `[]` should be removed.